lib/perl/Genome/Model/Build/ImportedAnnotation.t

```diff
@@ -20,6 +21,9 @@ my @versions = ('12_34', '56_78');
 my $ann_pp = Genome::ProcessingProfile::ImportedAnnotation->create(name => 'test_ann_pp', annotation_source => 'test_source');
 my $data_dir = File::Temp::tempdir('ImportedAnnotationTest-XXXXX', CLEANUP => 1, TMPDIR => 1);
 
+Genome::Test::Factory::AnalysisProject->setup_system_analysis_project;
```

Comment:
The system AnP is only automatically applied when going through the `define` command interface, so it has to manually attached here.

I wanted to test that it works with any arbitrary AnP.  It should be completely equivalent, though, so perhaps there's no particularly good reason to use a different one.